invenio_app_ils/ui/src/invenio_app_ils/common/components/DateRangePicker/DateRangePicker.js

```diff
@@ -7,17 +7,15 @@ export class DateRangePicker extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      fromDate: '',
-      toDate: '',
+      fromDate: props.defaultStart ? props.defaultStart : '',
+      toDate: props.defaultEnd ? props.defaultEnd : '',
     };
   }
 
   handleDateChange = (event, { name, value }) => {
-    const fromDate = name === 'fromDate' ? value : this.state.fromDate;
-    const toDate = name === 'toDate' ? value : this.state.toDate;
     this.setState({ [name]: value });
     if (this.props.handleDateChange) {
-      this.props.handleDateChange(fromDate, toDate);
+      this.props.handleDateChange({ name, value });
```

Comment:
Did you mean to do
```js
this.props.handleDateChange(name, value);
```
here since `LoanRequestForm` expects this?
```js
handleDateChange = (name, value) => {
```